tests/test_registration.py

```diff
@@ -13,18 +13,17 @@ def setUpClass(cls):
         TEST_DIR = os.path.abspath(os.path.dirname(__file__))
         os.chdir(os.path.join(TEST_DIR, "testdata"))
 
-    def test_feature_detector(self):
+    def test_feature_detector_mask(self):
```

Comment:
as you see my testing is not that extensive and structured. But thanks for also add some testing code :)

Reply:
I tried at least !